frontend/src/context/useHomeLocationQuery.js

```diff
@@ -0,0 +1,18 @@
+import { useEffect } from 'react';
+import { useLocationContext } from './useCurrentLocation';
+
+const useHomeLocationQuery = () => {
+  const { setCurrentLocation, coordinates } = useLocationContext()
+
+  const mount = () => {
+    setCurrentLocation();
+    const unmount = () => console.log('unmounted');
+    return unmount
+  }
+
+  useEffect(mount, []);
+
+  return coordinates
+};
+
+export default useHomeLocationQuery;
```

Comment:
What is that file for and why don't you just use the `useLocationContext` in map?